kolibri/plugins/coach/assets/src/views/ClassLearnersListPage.vue

```diff
@@ -133,6 +141,16 @@
         }
         return backRoute;
       },
+      learnerHasInsufficientStorage() {
+        for (const learner in this.learnerMap) {
+          if (this.classSyncStatusList[learner]) {
+            if (this.classSyncStatusList[learner].status === SyncStatus.INSUFFICIENT_STORAGE) {
```

Comment:
This `if` statement can be merged with the first `if` statement to avoid deep nesting. This approach would have made more sense if the inner `if` statement had an `else` but in our case there is none so we can use this opportunity to keep the logic more concise. Also, depending on your preference, you might want to assign the `this.classSyncStatusList[learn]` to a variable and use it instead within the `if` statement.

Reply:
That makes sense, I will try that and update the PR